source/renderer/app/stores/SidebarStore.js

```diff
@@ -19,6 +31,8 @@ export default class SidebarStore extends Store {
       this._onActivateSidebarCategory
     );
     sidebarActions.walletSelected.listen(this._onWalletSelected);
+    sidebarActions.changeWalletSortType.listen(this._onChangeWalletSortType);
+    sidebarActions.searchValueUpdated.listen(this._onSearchValueUpdated);
```

Comment:
We discussed in one of our dev calls that we figured that we want to move away from using the "Action" pattern because it doesn't add any benefits over public class methods. So in this case you could remove both actions and also remove the `_` in front of the class methods + call them directly from within the components.